nss-cohort-36/bangazon-api-purely-magic-ecomm-sprint-1
# Caroline Models and Fixtures

# Description

corrected imports on models to accept foreign keys


Fixes # (issue)

corrected imports on model 


## Type of change
Please delete options that are not relevant.


# Testing Instructions

commit changes to branch
checkout to master
git fetch --all
git checkout carolineMigrations
run code .
Models should have correct imports to accept classes on foreign keys and meta should be labeled correctly



# Checklist:
